src/game.js

```diff
@@ -0,0 +1,71 @@
+class Game {
+  constructor() {
+    this.round = 0;
+    this.score = 0;
+    this.frames = [];
+    this.isSpare = false;
```

Comment:
It's a clever idea to create variables for spare/strike - I wish I had thought of that